node/pkg/accountant/ntt_config.go

```diff
@@ -0,0 +1,68 @@
+package accountant
+
+import (
+	"fmt"
+
+	"github.com/certusone/wormhole/node/pkg/common"
+	"github.com/wormhole-foundation/wormhole/sdk"
+	"github.com/wormhole-foundation/wormhole/sdk/vaa"
+)
+
+type emitterConfigEntry struct {
+	chainId vaa.ChainID
+	addr    string
+	logOnly bool
+}
+
+type emitterConfig []emitterConfigEntry
+
+// nttGetEmitters returns the set of direct NTT and AR emitters based on the environment passed in.
+func nttGetEmitters(env common.Environment) (validEmitters, validEmitters, error) {
+	var directEmitterConfig emitterConfig
+	arEmitterConfig := sdk.KnownAutomaticRelayerEmitters
+	if env == common.MainNet {
+		directEmitterConfig = emitterConfig{}
+	} else if env == common.TestNet {
+		directEmitterConfig = emitterConfig{}
+		arEmitterConfig = sdk.KnownTestnetAutomaticRelayerEmitters
+	} else {
+		// Every other environment uses the devnet ones.
+		directEmitterConfig = emitterConfig{
+			{chainId: vaa.ChainIDEthereum, addr: "000000000000000000000000855FA758c77D68a04990E992aA4dcdeF899F654A"},
```

Comment:
I assume duplicate chains and identical addrs is correct for devnet?

Reply:
Yes, this is what's needed for the end to end tests.

Reply:
yes, we use two different wallets on different chains to simulate two different networks of NTTs (as this is capable of tracking many NTTs with the one NTT accountant)